tsdb/tsdbutil/buffer.go

```diff
@@ -215,8 +215,7 @@ func (r *sampleRing) last() (int64, float64, bool) {
 	if r.l == 0 {
 		return 0, 0, false
 	}
-	s := r.buf[r.i]
-	return s.t, s.v, true
+	return r.buf[r.i].t, r.buf[r.i].v, true
```

Comment:
The previous version was easier to read, this is a bit line noisy

Reply:
OK. I will change it back.